### ISSUE TITLE
Test snap command for minio server

### ISSUE THREAD

**Issue by deekoder** [minio/minio#5135]
Need to check snap related documentation for minio server. How and where are the keys printed.
Is our documentation around snap correct? @nitisht  can you verify this.

**Comment by deekoder** [comment]
@nitisht  can you review the documentation. How will people who intall via snap know what the keys are?

**Comment by nitisht** [comment]
@deekoder I actually tried running with snap and looks like server was not running. I need to check further. Will update this today

**Comment by harshavardhana** [comment]
The build for snap is not compatible with the libc we are using right now. I think only LTS will work @nitisht

**Comment by nitisht** [comment]
I still get the same on M2

```sh
minio@minio2:$ sudo snap install minio --edge
minio (edge) master from 'minio-dev' installed
minio@minio2:$ minio
minio: relocation error: /snap/minio/429/lib/x86_64-linux-gnu/libpthread.so.0: symbol __mmap, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
minio@minio2:$ minio server /tmp/test
minio: relocation error: /snap/minio/429/lib/x86_64-linux-gnu/libpthread.so.0: symbol __mmap, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
minio@minio2:$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
```

**Comment by harshavardhana** [comment]
Fixed by #5266 and #5256
